packages/fhir-converter/package.json

```diff
@@ -5,7 +5,8 @@
   "main": "src/index.js",
   "private": true,
   "scripts": {
-    "test": "nyc node --experimental-worker node_modules/mocha/bin/_mocha --reporter-options configFile=mocha.config.json --exit",
+    "test": "node --experimental-worker node_modules/mocha/bin/_mocha --reporter-options configFile=mocha.config.json --exit",
+    "coverage": "nyc run test",
```

Comment:
`nyc` was error'ing to execute tests b/c of low coverage, so separate the command